android/java/org/chromium/chrome/browser/crypto_wallet/activities/BuySendSwapActivity.java

```diff
@@ -971,18 +971,11 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
                             });
                 }
             } else if (mActivityType == ActivityType.BUY) {
-                assert mBlockchainRegistry != null;
-                String symbol = AssetUtils.mapToRampNetworkSymbol(mCurrentBlockchainToken);
```

Comment:
`mapToRampNetworkSymbol` is used only for `RAMP_NETWORK`. Moved the logic into `BuyModel` class.